src/neo/Ledger/StorageItem.cs

```diff
@@ -86,7 +87,7 @@ public T GetInteroperable<T>() where T : IInteroperable, new()
             if (cache is null)
             {
                 var interoperable = new T();
-                interoperable.FromStackItem(BinarySerializer.Deserialize(value, 16, 34));
+                interoperable.FromStackItem(BinarySerializer.Deserialize(value, ExecutionEngineLimits.Default.MaxStackSize, ExecutionEngineLimits.Default.MaxItemSize));
```

Comment:
Do we still need it?

Reply:
I think that it's better to preserve the vm limits.